src/Components/Classes/Form/index.jsx

```diff
@@ -369,18 +323,17 @@ const Form = () => {
           <h2>{id ? 'Edit' : 'Add'}</h2>
           {showSuccessModal && (
```

Comment:
The success modal it's not working, you have to use it when it redirects you in the index